src/databases/MFEM/CMakeLists.txt

```diff
@@ -28,6 +28,7 @@ JSONRoot.C
 
 INCLUDE_DIRECTORIES(
 ${CMAKE_CURRENT_SOURCE_DIR}
+${CONDUIT_INCLUDE_DIR}
```

Comment:
This shouldn't be necessary. It should be handled by the VISIT_MFEM_INCDEP entry in the host's config-site file and the logic I added to SetUpThirdParty.cmake to handle INCDEP entries. What was generated as MFEM_INCLUDE_DIR in your CMakeCache.txt?

Reply:
I see this: MFEM_INCLUDE_DIR:PATH=/home/bjw/Development/thirdparty_shared/3.0.0/mfem/3.4/linux-x86_64_gcc-7.4/include;/home/bjw/Development/thirdparty_shared/3.0.0/conduit/v0.4.0/linux-x86_64_gcc-7.4/include/conduit

I will try rebuilding on develop to see if the MFEM problem persists.

Reply:
Another build of develop worked cleanly. I can't explain it.